wry/AMTKVM.py

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/env python2
+
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import wsmanModule
+import json
+
+'''
+Created on 4 Jul 2017
+
+@author: adrian
+'''
+
+AMT_KVM_ENABLEMENT_MAP = {
+    2: (True, 'Enabled In Use'),
+    6: (True, 'Enabled'),
+    3: (False, 'Disabled'),
+}
+
+
+#class EnablementMap(object):
```

Comment:
Don't commit this, if it's not ready or used?

Reply:
It should have been removed. It has now.